lib/src/main/kotlin/org/konigsoftware/kontext/KonigKontext.kt

```diff
@@ -0,0 +1,87 @@
+package org.konigsoftware.kontext
+
+import com.google.protobuf.Message
+import io.grpc.Context
+import kotlin.reflect.KClass
+import kotlin.reflect.full.functions
+import kotlin.reflect.jvm.javaMethod
+
+class KonigKontext<KontextType>(
```

Comment:
Something about the naming here and/or storing an instance of this in a global variable confused me a bit

I interpreted the type name to mean this is an instance of an actual request-specific context, but it's actually more of a typed wrapper to encode/decode and fetch/set a request context, which is of type `KontextType` ?

Reply:
Addressed by adding KonigKontextKey